scripts/local_vault_setup.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -e
+vaultdir="$HOME/vault"
+mkdir -p $vaultdir
+if [ ! "$(command -v vault)" ]; then
+echo "downloading Vault for Linux"
```

Comment:
by convention, you may want to consider formatting these lines by indenting within the if statement